pom.xml

```diff
@@ -414,6 +414,7 @@
                 <configuration>
                     <enableAssertions>true</enableAssertions>
                     <trimStackTrace>false</trimStackTrace>
+                    <rerunFailingTestsCount>3</rerunFailingTestsCount>
```

Comment:
We should add the same configuration for the integration-tests below in the `maven-failsafe-plugin`, shouldn't we?
Link to docs: https://maven.apache.org/surefire/maven-failsafe-plugin/examples/rerun-failing-tests.html